app/kumactl/pkg/cmd/root_context.go

```diff
@@ -103,15 +84,11 @@ func DefaultRootContext() *RootContext {
 }
 
 func (rc *RootContext) TypeForArg(arg string) (core_model.ResourceType, error) {
-	typ, ok := rc.TypeArgs[arg]
+	typ, ok := entities.ByName[arg]
```

Comment:
IIUC, this needs to skip insight type that users should not delete.

Reply:
I've done this with a better error message just down there